src/blocks/recommendations/README.md

```diff
@@ -0,0 +1,52 @@
+# Parse.ly Recommendations Block
+
+This block is designed to showcase links to content on your site as provided by the [Parse.ly `/related` API endpoint](https://www.parse.ly/help/api/recommendations#get-related).
+
+You can use the Parse.ly Recommendations Block wherever you use the WordPress Block Editor ("Gutenberg") to build your site. That means that you can leverage this feature in `Posts`, `Pages`, or nearly any other custom post type. You can also leverage it in the Full Site Editor's template editor [slated to debut with WordPress version 5.8](https://make.wordpress.org/core/2021/06/16/introducing-the-template-editor-in-wordpress-5-8/) to add it to various layout elements (e.g. a Header, Footer, or Sidebar).
```

Comment:
I went ahead and replaced "Gutenberg" with "WordPress Block Editor" in the Recommendations Block's code.

Concerning the bit about FSE, this will have to be revised anyway, as the Block is currently not stable in FSE mode. I will leave this open until this is done.